.github/ISSUE_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,44 @@
+---
+name: New Component
+about: Interested in contributing a new component to Fabric? This template includes necessary information to get started, and steps to completion
+
+---
+<!-- Use this template for new components or new component varients -->
+
+## Component Details
+
+#### Exports
+<!-- What will be included in index.ts file -->
+
+#### Imports
+<!-- What other components/modules will your component be using -->
+
+#### Intended Package
+<!-- OUFR/New Package/Charting etc... -->
+
+#### Code mockup/example
+<!-- List/describe all expected props -->
```

Comment:
This probably needs an example table. It probably also needs to reference the props naming guidelines.